symengine/lib/symengine_wrapper.pyx

```diff
@@ -462,7 +463,6 @@ cdef class Basic(object):
         return Basic._richcmp_(A, B, op)
 
     def _richcmp_(Basic A, Basic B, int op):
-        from sympy import Rel
```

Comment:
I'm fine with that. We'll be removing any implicit sympy conversion in the future, because it's slow.
With an inline import, we are removing the dependency of sympy for other operations.